frontend/javascripts/oxalis/model/actions/volumetracing_actions.ts

```diff
@@ -261,3 +279,6 @@ export const dispatchFloodfillAsync = async (
   dispatch(action);
   await readyDeferred.promise();
 };
+export const setMappingisEditableAction = (): SetMappingIsEditableAction => ({
```

Comment:
:eagle: :eyes: 

I would say it's rather likely that in the near future it will be possible to switch from an editable mapping to another (non-editable) one (and back again). The boolean parameter can be introduced then. So I'd vote to keep this as-is.